lib/ledger-node-operations.js

```diff
@@ -52,7 +81,20 @@ module.exports = class LedgerNodeOperations {
       config: callback => ledgerNode.config.get(callback),
       validate: ['config', (results, callback) => {
         const config = results.config;
+        const context = operation['@context'];
```

Comment:
Need to make sure `operation` is a non-null object too, right?

Reply:
We check that here https://github.com/digitalbazaar/bedrock-ledger-node/pull/20/files#diff-0f55df37610acaf177e9815d9667c70eR15